Surf MVP and Coordinators Application.xctemplate/Storyboard/Makefile

```
@@ -0,0 +1,101 @@
+init:
+	# Install bundler if not installed
+	if ! gem spec bundler > /dev/null 2>&1; then\
+  		echo "bundler gem is not installed! Run gem install bundler to install it";\
+  		-sudo gem install bundler;\
```

Нужно версию бандлера проставлять явно
```suggestion
  		-sudo gem install bundler -v "1.17.3";\
```